modules/nf-core/sourmash/compare/main.nf

```diff
@@ -0,0 +1,39 @@
+process SOURMASH_COMPARE {
+    label 'process_low'
+
+    conda (params.enable_conda ? "bioconda::sourmash=4.5.0" : null)
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/sourmash:4.5.0--hdfd78af_0':
+        'quay.io/biocontainers/sourmash:4.5.0--hdfd78af_0' }"
+
+    input:
+    path signatures
+    val save_numpy_matrix
```

Comment:
Missing optional input file [^1] for `--from-file` flag
```suggestion
    path(file_list)
    val save_numpy_matrix
```

[^1]: https://nf-co.re/docs/contributing/modules#inputoutput-options point 1